### ISSUE TITLE
Why the suggestions has no whitespace?

### ISSUE THREAD

**Issue by devcosta** [tuantle/react-native-search-header#23]
SearchHeader component:

`<SearchHeader
          ref = {(searchHeader) => {
              this.searchHeader = searchHeader;
          }}
          onGetAutocompletions ={this.search}
        />`

My function:
`search = (text) => {
      return ["This is a simple test"];
    }
  }`

Console:
`Array [
      "This is a simple test"
]`

"react-native": "~0.54.1",
"react-native-search-header": "^0.2.6"
 "expo": "^26.0.0" ;)

Android: 6.0:
![screenshot_20180524-181247](https://user-images.githubusercontent.com/3438209/40514299-fdc1a56e-5f7e-11e8-9541-3f01e0ce4cdb.png)

ty guys ;)

**Comment by tuantle** [comment]
You can try and set the style of the suggestion view directly and see if it fix your issue.

Change `maxHeight: DEVICE_HEIGHT / 2` to the height value that you want.

Set the style of suggestion view from flexShrink:1 to flexGrow:1 would work too.

```
    suggestion: {
        ...DEFAULT_DROP_SHADOW_STYLE,
        flexShrink: 1,
        flexDirection: `column`,
        alignItems: `stretch`,
        justifyContent: `center`,
        maxHeight: DEVICE_HEIGHT / 2,
        paddingLeft: 12,
        marginVertical: 6,
        backgroundColor: `#fdfdfd`
    }
```

**Comment by tuantle** [comment]
Oh so if you look at line 557 in node_modules/react-native-search-header/search-header.js

`...new Set(autocompleteTexts.filter((text) => typeof text === `string`).map((text) => text.replace(/\s/g, ``)))`

I have the trailing white space remove with this regex `/\s/g, ```. It works on iOS, but looks like it is causing issue on android? 

You can remove `.map((text) => text.replace(/\s/g, ``)))` and see if it fixes your issue.

**Comment by tuantle** [comment]
I fixed this in new version 0.2.7

**Comment by devcosta** [comment]
Ty man, perfect ;)
